dictionary/templates/dictionary/search.html

```diff
@@ -6,14 +6,16 @@
 
 {% block body %}
 <div class="container">
-    <form method="POST" action="" >
+    <form method="GET" action="" >
         {% csrf_token %}
         {% bootstrap_form form %}
         <button type="submit" class="btn btn-primary">Search</button>
     </form>
+</div>
 
     <hr />
-    {% if results%}
+<div class="container">
```

Comment:
I believe that technically you should only have one bootstrap `container` (or `container-fluid`) in a document and then everything that should be bootstrapped goes inside with `row`, `col-xs-12`, etc.